workflows/sock-shop-demo/workflow.yaml

```diff
@@ -0,0 +1,182 @@
+      inputs:
+        artifacts:
+          - name: install-experiment
+            path: /tmp/pod-delete.yaml
+            raw:
+              data: |
+                apiVersion: litmuschaos.io/v1alpha1
+                description:
+                  message: |
+                    Deletes a pod belonging to a deployment/statefulset/daemonset
+                kind: ChaosExperiment
+                metadata:
+                  name: pod-delete
+                spec:
+                  definition:
+                    scope: Namespaced
+                    permissions:
+                      - apiGroups:
+                          - ""
+                          - "apps"
+                          - "batch"
+                          - "litmuschaos.io"
+                        resources:
+                          - "deployments"
+                          - "jobs"
+                          - "pods"
+                          - "pods/log"
+                          - "events"
+                          - "configmaps"
+                          - "chaosengines"
+                          - "chaosexperiments"
+                          - "chaosresults"
+                        verbs:
+                          - "create"
+                          - "list"
+                          - "get"
+                          - "patch"
+                          - "update"
+                          - "delete"
+                      - apiGroups:
+                          - ""
+                        resources: 
+                          - "nodes"
+                        verbs:
+                          - "get"
+                          - "list"
+                    image: "litmuschaos/go-runner:latest"
+                    imagePullPolicy: Always
+                    args:
+                    - -c
+                    - ./experiments -name pod-delete
+                    command:
+                    - /bin/bash
+                    env:
+                    - name: TOTAL_CHAOS_DURATION
+                      value: '15'
+                    # Period to wait before and after injection of chaos in sec
+                    - name: RAMP_TIME
+                      value: ''
+                    # provide the kill count
+                    - name: KILL_COUNT
+                      value: ''
+                    - name: FORCE
+                      value: 'true'
+                    - name: CHAOS_INTERVAL
+                      value: '5'
+                    - name: LIB
+                      value: 'litmus'    
+                    labels:
+                      name: pod-delete
+      container:
+        image: lachlanevenson/k8s-kubectl
+        command: [sh, -c]
+        args:
+          [
+            "kubectl apply -f /tmp/pod-delete.yaml -n {{workflow.parameters.adminModeNamespace}}",
+          ]
+    
+    - name: load-test
+      container:
+        image: litmuschaos/litmus-app-deployer:latest
+        args: ["-namespace=loadtest"]
```

Comment:
Are you tuning load-test and sock-shop run based on the value of namespace?

Reply:
May I use a flag like a bool type? 
Is that ok to have a namespace flag only? It is also working as a flag only.

Reply:
deployerType/typeName flag can be used, which can take [weak, resilient,load-test] values and test-tool code can select the corresponding file path based on this flag?
namespace flag can still be there. You can set some default value based on usecase(sock-shop for sock-shop and loadtest for load-test)

Reply:
We have merged the PR on test-tools: https://github.com/litmuschaos/test-tools/pull/145 that is dependent on the namespace name to launch the right manifest. This is something we should change in the next release. Resolving for now.